added custom location for component directory (#131)
hmcts/cnp-azuredevops-libraries: README.md

> see [example in aks-cft-deploy repo](https://github.com/hmcts/aks-cft-deploy/blob/main/azure-pipelines.yml)

> ### components file location

Reviewer: ```suggestion
### Override components directory
```

> On repositories, it may not be possible to have components folder directy under the root of the repository. In such case, the `baseDirectory` option can be used so specify the location of the components folder.

Reviewer: ```suggestion
In a monorepo, it may be needed to override the components folder if there's multiple applications that are built from the same repository. You can pass the `baseDirectory` option to specify the location of the components folder.
```

```yaml
baseDirectory: "terraform/network/components"
```

> > see [example in oracle-azure-infrastructure repo](https://github.com/hmcts/oracle-azure-infrastructure/blob/DTSPO-9224-pipeline-refactor/terraform/network/azure-pipelines.yml)

Reviewer: 🚫 do not link to non master references it will be deleted after merge, 

You could possibly just remove this link as external repository links are risky, someone might refactor this and not update this link, someone can easily search the HMCTS GitHub organisation to see how to use the parameter
```suggestion
> see [example in oracle-azure-infrastructure repo](https://github.com/hmcts/oracle-azure-infrastructure/blob/master/terraform/network/azure-pipelines.yml)
```

Author: Yupp - I was going to change it but wanted to make sure link works if you were going to press it :)
